package/system/uci/Makefile

```diff
@@ -76,6 +80,10 @@ define Package/uci/install
 	$(CP) ./files/* $(1)/
 endef
 
+CMAKE_HOST_OPTIONS += \
+	-DBUILD_LUA=OFF \
+	-DBUILD_STATIC=ON
```

Comment:
Why do you need this options?

Reply:
I thought it would be better to have a static build for uci on the host.
I'm just running a build and the compiler seems to be happy.
I still have to check if the dynamic libraries are found on the host on use.